### PR TITLE
Allow to seed from spheres, and to specify initial velocity

### PR DESCRIPTION
**Pull request by yuanming-hu**
For example, sometimes it would be helpful for Blender users to set the initial velocity of a water jet.

ezgif-3-0f83adbbb136

### CODE REVIEW

File: demo/demo_2d.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import taichi as ti
 import numpy as np
-import utils
```
[the remark this answers is not in the document]
Comment (yuanming-hu) [reply demo/demo_2d.py:3]: Thanks for pointing this out. Sorry I didn't realize `utils` are indirectly used in the script :-)

[the remark this answers is not in the document]
Comment (PavelBlend) [reply demo/demo_2d.py:3]: This is a kind of workaround with which you can import the engine module. Perhaps this solution is not the best. If you have a better idea, you can do it differently.

[the remark this answers is not in the document]
Comment (yuanming-hu) [reply demo/demo_2d.py:3]: Right, I used to suffer from these Python import issues. I don't have a better idea, to be honest...

**Comment by PavelBlend** [comment]
And I have a question:
Do you need to add spheres to the blender? Are these spheres intended for demo scenarios only? In a blender, you can create a sphere in the form of an object mesh.. I will add only initial velocity.

**Comment by yuanming-hu** [comment]
> Do you need to add spheres to the blender? Are these spheres intended for demo scenarios only?

That's for demo only. In Blender using meshes is probably better.

**Pull request merged**